### DISCUSSION TITLE
portal2

### DISCUSSION THREAD

**Discussion by weitao23333** [Gamuxorg/bbs#312] [游戏或网站运行反馈]
游戏名称：portal2

操作系统：*archlinux*

bug描述：*游戏无法启动*

终端运行输出 ：*
libGL error: MESA-LOADER: failed to open nouveau (search paths /usr/lib32/dri)
libGL error: failed to load driver: nouveau
libGL error: MESA-LOADER: failed to open swrast (search paths /usr/lib32/dri)
libGL error: failed to load driver: swrast
X Error of failed request:  GLXBadContext
  Major opcode of failed request:  152 (GLX)
  Minor opcode of failed request:  6 (X_GLXIsDirect)
  Serial number of failed request:  82
  Current serial number in output stream:  81
*

**Comment by hyh19962008** [comment]
请确保已安装32位mesa驱动，检查/usr/lib/i386-linux-gnu/dri 目录下是否存在错误信息中提到的文件，如有的话，可以在/usr/lib32/dri下创建对应文件的软链接。

Best

________________________________

**Comment by weitao23333** [comment]
/usr/lib/i386-linux-gnu/dri 目录下无文件，但驱动工作正常，而且可以运行portal1

**Comment by hyh19962008** [comment]
libgl1-mesa-dri:i386

Best

________________________________
